app/src/ui/add-repository/licenses.ts

```diff
@@ -74,6 +74,14 @@ function replaceTokens(
   return newBody
 }
 
+function ensureLineEndingSet(body: string) {
+  if (body.endsWith('\n')) {
```

Comment:
Using conditional `?` operator instead of this if conditional check makes it appear clean. Just thoughts :thinking: